core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreMainActivity.java

```diff
@@ -235,21 +238,21 @@ public void onFinish() {
   private ItemTouchHelper.Callback tabCallback = new ItemTouchHelper.Callback() {
     @Override
     public int getMovementFlags(@NonNull RecyclerView recyclerView,
-      @NonNull RecyclerView.ViewHolder viewHolder) {
+                                @NonNull RecyclerView.ViewHolder viewHolder) {
```

Comment:
these indentations are a bit odd, `ctrl+alt+l` should fix it. I generally recommend binding format to a macro (ctrl+s) that formats, optimises imports and saves all files